src/mcts/node.cc

```diff
@@ -153,11 +152,8 @@ void Edge::SetP(float p) {
 }
 
 float Edge::GetP() const {
-  // Reshift into place and set the assumed-set exponent bits.
   uint32_t tmp = (static_cast<uint32_t>(p_) << 12) | (3 << 28);
-  float ret;
-  std::memcpy(&ret, &tmp, sizeof(uint32_t));
-  return ret;
+  return *reinterpret_cast<float*>(&tmp);
```

Comment:
Some context why the memcpy was there: <https://discordapp.com/channels/425419482568196106/425419999096733706/471119745220083741>

Reply:
Yeah it's not allowed by type aliasing rules:
https://en.cppreference.com/w/cpp/language/reinterpret_cast

Reply:
Ohh ok, going to revert that commit.